package.json

```diff
@@ -11,7 +11,10 @@
 		"url": "https://sindresorhus.com"
 	},
 	"type": "module",
-	"exports": "./index.js",
+	"exports": {
```

Comment:
Does TypeScript pick up `index.d.ts` when using the `browser` type export? Or do we need to specify the type definition explicitly. I think the TS 4.7 blog post has some info about this.

Reply:
After a quick skim, it appears that it *might* help to also add `"types": "./index.d.ts"`.

https://devblogs.microsoft.com/typescript/announcing-typescript-4-7-beta/#package-json-exports-imports-and-self-referencing